cloudevents/tests/test_pydantic_events.py

```diff
@@ -372,12 +427,14 @@ def test_empty_data_binary_event(specversion, cloudevents_implementation):
 
 
 @pytest.mark.parametrize("specversion", ["1.0", "0.3"])
-def test_valid_structured_events(specversion, cloudevents_implementation):
+def test_valid_structured_events(
+    specversion: str, cloudevents_implementation: PydanticImplementation
+) -> None:
     # Test creating multiple cloud events
-    events_queue = []
+    events_queue: list[AnyPydanticCloudEvent] = []
     num_cloudevents = 30
     for i in range(num_cloudevents):
-        event = {
+        raw_event = {
```

Comment:
The issue here is that the second loop assigns the event var with a different type, causing a type error. Using a different name avoids this.